.coveragerc

```diff
@@ -696,4 +697,4 @@ exclude_lines =
 
     # Don't complain if tests don't hit defensive assertion code:
     raise AssertionError
-    raise NotImplementedError
\ No newline at end of file
+    raise NotImplementedError
```

Comment:
Revert this modification.

Reply:
Hi, thanks for the points. Not sure why it's showing this as added as the line was there, it just moved 1 line down.